src/uploader.c

```diff
@@ -370,7 +370,7 @@ static void after_create_bucket_entry(uv_work_t *work, int status)
         state->info->id = NULL;
         state->info->bucket_id = state->bucket_id;
         state->info->decrypted = true;
-        state->info->index = strdup(state->index);
+        state->info->index = state->index;
```

Comment:
This may have been a regression, `state->info->index = strdup(state->index);` is how this should be (and should go along with leaving the free to file->index in `storj_free_uploaded_file_info`.

Reply:
made changes per the comment